SUNET/eduid-front
# Implement eslint action

#### Description:
As agreed during the "work process discussion", I have installed "eslint" for making the code more consistent, avoiding bugs and specify the code structured. 

I have added "npm run lint" to "run-tests.yaml" so that CI will test for "eslint", but I have not set eslint rules yet, due to "ESLint errors" that would fail the build if not sorted. So I will make a separate PR where I will add the rules and fix the errors.


------
**To test running lint for this PR**
adding  
  ```
  "extends": [
           "eslint:recommended",
           "plugin:react/recommended"
    ],
```
**.eslintrc.js** ,  run **npm run lint** then you will get errors

------

**[future reference]** Installing ESLint steps

npm install --dev eslint
npm install --dev eslint-config-standard
npm install —dev eslint-plugin-node eslint-plugin-react

npx eslint --init

```
How would you like to use ESLint? 
❯ To check syntax and find problems?
What type of modules does your project use? … 
❯ JavaScript modules (import/export)
Which framework does your project use? … 
❯ React
Does your project use TypeScript? › 
❯ No
Where does your code run?
❯ Browser
 What format do you want your config file to be in?
❯ JavaScript
```

#### For reviewer:
- [ ] Read the above description
- [ ] Reviewed the code changes
- [ ] Navigate to the branch (pulled the latest version)
- [ ] Run the code and been able to execute the expected function
- [ ] Check any styling on both desktop and mobile sizes
